editor.planx.uk/src/@planx/components/FindProperty/Public/FindProperty.stories.tsx

```diff
@@ -1,7 +1,8 @@
 import { Meta, Story } from "@storybook/react/types-6-0";
 import React from "react";
 
-import FindProperty, { PropertyInformation } from "./";
+import PropertyInformation from "../../PropertyInformation/Public";
+import FindProperty from "./";
```

Comment:
I admittedly took a shortcut here by continuing to combine these components in the same "story" - will revisit/clean up in the future if that's okay (lots of bits of storybook are out of date, more FindProperty updates in the pipeline, etc)